apps/client/src/features/rundown/event-block/composite/EventBlockChip.tsx

```diff
@@ -75,3 +78,51 @@ function EventUntil(props: EventUntilProps) {
     </Tooltip>
   );
 }
+
+interface EventReportProps {
+  className: string;
+  id: string;
+  duration: number;
+}
+
+function EventReport(props: EventReportProps) {
+  const { className, id, duration } = props;
+  const { data } = useReport();
+  const currentReport: OntimeEventReport | undefined = data[id];
+
+  const [value, isOver] = useMemo(() => {
+    if (!currentReport) {
+      return [null, false];
+    }
+
+    const { startedAt, endedAt } = currentReport;
+    if (!startedAt || !endedAt) {
+      return [null, false];
+    }
+
+    const actualDuration = endedAt - startedAt;
+    const difference = actualDuration - duration;
+    const absDifference = Math.abs(difference);
+
+    if (absDifference < MILLIS_PER_SECOND) {
+      return ['==', false]; //TODO: how to symbolize ontime?
+    }
+
+    const isOver = difference > 0;
+
+    const value = `${isOver ? '+' : '-'}${formatDuration(Math.abs(difference), difference > 2 * MILLIS_PER_MINUTE)}`;
+    return [value, isOver];
+  }, [currentReport, duration]);
+
+  if (!value) {
+    return null;
+  }
+
+  return (
+    <Tooltip label='Difference from the expected duration' openDelay={tooltipDelayFast}>
```

Comment:
Maybe the tooltip for past events should explain the data like
- Event gained xx:xx time
- Event lost xx:xx time

I believe this may be safe since the past event times should no longer change

Reply:
i'm wondering if the words are clear
gained? 

did the event run shorter than expected so we gained some time back in a show running behind 
or
did the event it self gain time by taking to long

Reply:
> i'm wondering if the words are clear gained?
> 
> did the event run shorter than expected so we gained some time back in a show running behind or did the event it self gain time by taking to long

hahaha, wording things is difficult. I suggest we leave it as is and wait for user feedback.
Either way, I hope the user will see that this looks like the offset and connect the dots